src/main/java/com/google/cloud/tools/app/api/DefaultConfiguration.java

```diff
@@ -14,9 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.app.impl.config;
-
-import com.google.cloud.tools.app.api.Configuration;
+package com.google.cloud.tools.app.api;
```

Comment:
@meltsufin 

I think we made a mistake making this the same name as the gcloud 'app' command. gcloud is a cli that should be minimally verbose,  I think with package names in java we should be as descriptive as we can.   What do you think of changing from app to appengine?  This will also match the package naming in other tools better.